src/renderer/app-update-warning/app-update-warning.injectable.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import ipcRendererInjectable from "../app-paths/get-value-from-registered-channel/ipc-renderer/ipc-renderer.injectable";
```

Comment:
`ipcRendererInjectable` is a pretty low-level abstraction to be used in code like this. Use a new `MessageChannel` instead. There's also something called `SyncBox`.